.rubocop.yml

```diff
@@ -54,6 +56,8 @@ Layout/EmptyLinesAroundAttributeAccessor:
   Enabled: true
 Layout/EndAlignment:
   Severity: error
+Layout/EndOfLine:
+  Enabled: false
```

Comment:
Could we do EnforcedStyle: lf to use line feed everywhere? Maybe in a separate PR?

Reply:
We could use `EnforcedStyle: lf` in the current PR as well without affecting CI or including more file changes. All files are already checked out by git with native line-endings automatically.

But for contributors-from-Windows' sake, we may perhaps need to reconfigure `.gitattributes` file.

Reply:
Ok, I think we'd prefer to check in LF instead of CRLF. Can .gitattributes give us a per-repo way of saying "Windows needs CRLF so write them to disk but Git should remove the CR"? As a Windows user (or one in the past), I think you have the best understanding of how this should work.